sdk/include/priv/riscv.h

```diff
@@ -75,6 +75,7 @@ namespace priv
 	constexpr size_t MCAUSE_LOAD_PAGE_FAULT     = 13;
 	constexpr size_t MCAUSE_STORE_PAGE_FAULT    = 15;
 	constexpr size_t MCAUSE_THREAD_EXIT         = 24;
+	constexpr size_t MCAUSE_THREAD_INTERRUPT    = 25;
 	constexpr size_t MCAUSE_CHERI               = 28;
```

Comment:
This could be a separate PR but these constants are needed in error handlers so we should make them more publicly available in the SDK.

Reply:
This is a public heading (and this is why I don't like abbreviations in names of things: priv is short for 'privileged spec' not 'private').

Reply:
I guess there's nothing stopping users from include `priv/riscv.h` it just seems a bit weird. At any rate we should do this in relevant places in the examples/tests.